applications/nrf_desktop/bootloader_dfu.rst

```diff
@@ -189,24 +189,45 @@ For an example of a bootloader Kconfig configuration file defined by the applica
-  However, you cannot use this option if the secondary image slot uses external memory.
-* :kconfig:option:`SB_CONFIG_MCUBOOT_SIGNATURE_USING_KMU` - This option enables using Key Management Unit (KMU) to store keys for signature verification instead of compiling key data into the MCUboot bootloader image.
-  To use KMU, the public key must first be provisioned.
-  See the :ref:`ug_nrf54l_developing_provision_kmu` documentation for details.
+Bootloader features that are specific to the nRF54L and the nRF54H series are listed in their respective subsections below.
 
-  .. note::
-     To use automatic provisioning, enable the :kconfig:option:`SB_CONFIG_MCUBOOT_GENERATE_DEFAULT_KMU_KEYFILE` sysbuild Kconfig option.
-     This option enables generating a default :file:`keyfile.json` file during the build process based on the input file provided by the :kconfig:option:`SB_CONFIG_BOOT_SIGNATURE_KEY_FILE` sysbuild Kconfig option.
-     The automatic provisioning is only performed if the west flash command is executed with the ``--erase`` or ``--recover`` flag.
+MCUboot bootloader features specific to nRF54L series
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The nRF54L devices support the use of the Key Management Unit (KMU) to store keys for signature verification instead of compiling key data into the MCUboot bootloader image.
+To use KMU in the MCUboot bootloader, enable the :kconfig:option:`SB_CONFIG_MCUBOOT_SIGNATURE_USING_KMU` sysbuild Kconfig option.
+You must also make sure to provision the public key to your target device before running the firmware.
+See the :ref:`ug_nrf54l_developing_provision_kmu` documentation for details.
+
+.. note::
+    To use automatic provisioning, enable the :kconfig:option:`SB_CONFIG_MCUBOOT_GENERATE_DEFAULT_KMU_KEYFILE` sysbuild Kconfig option.
+    This option enables generating a default :file:`keyfile.json` file during the build process based on the input file provided by the :kconfig:option:`SB_CONFIG_BOOT_SIGNATURE_KEY_FILE` sysbuild Kconfig option.
+    The automatic provisioning is only performed if the west flash command is executed with the ``--erase`` or ``--recover`` flag.
+
+MCUboot bootloader features specific to nRF54H series
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+On the nRF54H devices, the MCUboot direct-xip mode uses a merged image slot that combines both application and radio core images.
+The merged binary size is the sum of the application image, the radio core image, and the padding between them.
+The merged image slot configuration is indicated by the :kconfig:option:`SB_CONFIG_MCUBOOT_SIGN_MERGED_BINARY` sysbuild Kconfig option.
+
+.. note::
+   Because padding is added between the application and radio core images, the merged binary ends up including the unused NVM from the application image partition.
```

Comment:
Consider informing that you can tailor partition size of the application image to the image size to reduce size of the padding and make the combined update image smaller.